### PR TITLE
Feature / TEC-3653 improved ui general tab

### PR DESCRIPTION
**Pull request by alliuca**
**Description:** Updates the Customizer UI to include headings and other copy changes

**Ticket:** https://moderntribe.atlassian.net/browse/TEC-3653

**Screenshots:**

Screen Shot 2020-11-25 at 12 39 51 PM

### CODE REVIEW

File: src/Tribe/Customizer/Global_Elements.php [hunk 1]
```diff
@@ -476,49 +478,68 @@ public function setup() {
 	public function register_settings( WP_Customize_Section $section, WP_Customize_Manager $manager ) {
 		$customizer = Tribe__Customizer::instance();
 
+		// Add an heading that is a Control only in name: it does not, actulally, control or save any setting.
+		$manager->add_control( new Heading(
```
Comment (tr1b0t) [inline src/Tribe/Customizer/Global_Elements.php:482]: :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line

<sub>hash: d34fef40b1affa17769f222bcea53d2a</sub>

File: src/Tribe/Customizer/Global_Elements.php [hunk 2]
```diff
@@ -476,49 +478,68 @@ public function setup() {
...
+			$manager,
+			$customizer->get_setting_name( 'accent_color_heading', $section ),
+			[
+				'label'   => esc_html__( 'Set Font Colors', 'the-events-calendar' ),
+				'section' => $section->id,
+			]
+		) );
```
Comment (tr1b0t) [inline src/Tribe/Customizer/Global_Elements.php:489]: :no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: 7cc10f337560357a34ba0251b80a7b88</sub>

File: src/Tribe/Customizer/Global_Elements.php [hunk 3]
```diff
@@ -476,49 +478,68 @@ public function setup() {
...
+
 		$manager->add_setting(
 			$customizer->get_setting_name( 'accent_color', $section ),
-			array(
-				'default'              => '#334AFF',
-				'type'                 => 'option',
-
+			[
+				'default' => '#334AFF',
```
Comment (tr1b0t) [inline src/Tribe/Customizer/Global_Elements.php:494]: :warning: Array double arrow not aligned correctly; expected 14 space(s) between "'default'" and double arrow, but found 1.

<sub>hash: b958272c6ee9bd6289b5c5e5929e3d4d</sub>

File: src/Tribe/Customizer/Global_Elements.php [hunk 4]
```diff
@@ -476,49 +478,68 @@ public function setup() {
...
+				'type'    => 'option',
```
Comment (tr1b0t) [inline src/Tribe/Customizer/Global_Elements.php:495]: :warning: Array double arrow not aligned correctly; expected 17 space(s) between "'type'" and double arrow, but found 4.

<sub>hash: 696141f4c84fd79c598113bc7c6c2c15</sub>

File: src/Tribe/Customizer/Global_Elements.php [hunk 5]
```diff
@@ -476,49 +478,68 @@ public function setup() {
...
 		$manager->add_control(
 			new WP_Customize_Color_Control(
 				$manager,
 				$customizer->get_setting_name( 'accent_color', $section ),
-				array(
-					'label'   => esc_html__( 'Accent Color', 'the-events-calendar' ),
-					'section' => $section->id,
-				)
+				[
+					'label'       => esc_html__( 'Accent Color', 'the-events-calendar' ),
```
Comment (tr1b0t) [inline src/Tribe/Customizer/Global_Elements.php:506]: :warning: Array double arrow not aligned correctly; expected 3 space(s) between "'label'" and double arrow, but found 7.

<sub>hash: a3f6673d9e7acd5dce1f72368f449005</sub>

File: src/Tribe/Customizer/Global_Elements.php [hunk 6]
```diff
@@ -476,49 +478,68 @@ public function setup() {
...
+					'section'     => $section->id,
```
Comment (tr1b0t) [inline src/Tribe/Customizer/Global_Elements.php:507]: :warning: Array double arrow not aligned correctly; expected 1 space(s) between "'section'" and double arrow, but found 5.

<sub>hash: 045e4e33a1138c5535bd4ce9519652b1</sub>

File: src/Tribe/Customizer/Global_Elements.php [hunk 7]
```diff
@@ -476,49 +478,68 @@ public function setup() {
...
 				'type'                 => 'option',
-
 				'sanitize_callback'    => 'sanitize_hex_color',
 				'sanitize_js_callback' => 'maybe_hash_hex_color',
-			)
+			]
 		);
 
+		// Add an heading that is a Control only in name: it does not, actulally, control or save any setting.
+		$manager->add_control( new Heading(
```
Comment (tr1b0t) [inline src/Tribe/Customizer/Global_Elements.php:525]: :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line

<sub>hash: cb7fd3ec45032705c0e01257e0d910d1</sub>

File: src/Tribe/Customizer/Global_Elements.php [hunk 8]
```diff
@@ -476,49 +478,68 @@ public function setup() {
...
+			$manager,
+			$customizer->get_setting_name( 'link_color_heading', $section ),
+			[
+				'label'   => esc_html__( 'Adjust Appearance', 'the-events-calendar' ),
+				'section' => $section->id,
+			]
+		) );
```
Comment (tr1b0t) [inline src/Tribe/Customizer/Global_Elements.php:532]: :no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: c0c0446606c23b7ff7425afad4727530</sub>

File: src/Tribe/Customizer/Global_Elements.php [hunk 9]
```diff
@@ -476,49 +478,68 @@ public function setup() {
...
 			new WP_Customize_Color_Control(
 				$manager,
 				$customizer->get_setting_name( 'link_color', $section ),
-				array(
-					'label'   => esc_html__( 'Link Color', 'the-events-calendar' ),
-					'section' => $section->id,
-				)
+				[
+					'label'       => esc_html__( 'Links', 'the-events-calendar' ),
+					'description'	=> esc_html__( 'For displayed URLs', 'the-events-calendar'),
```
Comment (tr1b0t) [inline src/Tribe/Customizer/Global_Elements.php:540]: * :no_entry_sign: Spaces must be used for mid-line alignment; tabs are not allowed
* :warning: Array double arrow not aligned correctly; expected 1 space(s) between "'description'" and double arrow, but found 3.
* :no_entry_sign: Expected 1 spaces before closing parenthesis; 0 found
* :no_entry_sign: There must be a single space before the last parenthesis

<sub>hash: 8c009a379fc86cc6e67c3c79e924ae93</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 7 error(s)
:warning: 5 warning(s)

**Comment by paulmskim** [comment]
@alliuca I restarted tests as they were failing, let's see if it works, otherwise we'll need to figure out what's going on

**Comment by alliuca** [comment]
@paulmskim great, it does seem to work now, thanks

**Review by paulmskim — APPROVED** [review]

**Pull request merged**